exporter/clickhouselogsexporter/config.go

```diff
@@ -31,10 +31,8 @@ type Config struct {
 	// DSN is the ClickHouse server Data Source Name.
 	// For tcp protocol reference: [ClickHouse/clickhouse-go#dsn](https://github.com/ClickHouse/clickhouse-go#dsn).
 	// For http protocol reference: [mailru/go-clickhouse/#dsn](https://github.com/mailru/go-clickhouse/#dsn).
-	DSN string `mapstructure:"dsn"`
-	// Docker Multi Node Cluster is a flag to enable the docker multi node cluster. Default is false.
-	DockerMultiNodeCluster bool `mapstructure:"docker_multi_node_cluster" default:"false"`
```

Comment:
I don't see `DockerMultiNodeCluster` used anywhere

Reply:
It is being used [here](https://github.com/SigNoz/signoz/blob/develop/deploy/docker/clickhouse-setup/docker-compose-minimal.yaml#L265). And I see the same being mentioned in multiple places in our docs.

If you think it is no longer relevant, please remove the reference from SigNoz and `signoz-web` repositories.

Reply:
When I say it is not used anywhere, I mean it is not used in the code. Yes, I will remove otherwise, the collector won't start.

Reply:
Got it 👍